packages/backend/src/app/ee/authentication/enterprise-local-authn/enterprise-local-authn-controller.ts

```diff
@@ -2,14 +2,26 @@ import { FastifyPluginAsyncTypebox } from '@fastify/type-provider-typebox'
 import { enterpriseLocalAuthnService } from './enterprise-local-authn-service'
 import { ResetPasswordRequestBody, VerifyEmailRequestBody } from '@activepieces/ee-shared'
 import { ALL_PRINICPAL_TYPES } from '@activepieces/shared'
+import { eventsHooks } from '../../../helper/audit-events'
+import { ApplicationEventName } from '@activepieces/ee-shared'
 
 export const enterpriseLocalAuthnController: FastifyPluginAsyncTypebox = async (app) => {
     app.post('/verify-email', VerifyEmailRequest, async (req) => {
         await enterpriseLocalAuthnService.verifyEmail(req.body)
+        await eventsHooks.get().send(req, {
+            action: ApplicationEventName.VERIFIED_EMAIL,
+            userId: req.body.userId,
+            projectId: req.principal.projectId,
+        })
```

Comment:
I think adding `await` here was by mistake, right? I think no need to wait for this promise as its return value is ignored anyway. Just make sure to append `.catch`.

Reply:
send is not longer async, and implemented rejectpromise handler and have it inside the audit service.